src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -271,28 +271,16 @@
                     @foreach (var linksConfiguration in Model.LinksConfiguration)
                     {
                         var pageId = Model.PageId;
-                        var first = true;
                         @foreach (var link in linksConfiguration.Links)
                         {
 
                             if (!string.IsNullOrEmpty(link.LinkPageId))
                             {
                                 pageId = link.LinkPageId;
                             }
-
-                            if (first && Model.SectionId != 1)
-                            {
-
-                                <a class="govuk-link das-inline-button-link govuk-link--no-visited-state" asp-action="@link.Action" asp-controller="@link.Controller" asp-route-applicationId="@Model.ApplicationId" asp-route-sequenceId="@Model.SequenceId" asp-route-sectionId="@Model.SectionId" asp-route-pageId="@pageId">@link.Caption</a>
-
-                            }
-                            else
-                            {
-                                first = false;
-                                <p class="govuk-body">
-                                    <a class="govuk-link" asp-action="@link.Action" asp-controller="@link.Controller" asp-route-applicationId="@Model.ApplicationId" asp-route-sequenceId="@Model.SequenceId" asp-route-sectionId="@Model.SectionId" asp-route-pageId="@pageId">@link.Caption</a>
-                                </p>
```

Comment:
This existing markup is correct (the paragraph tag wrapping the link tag) for the link below the button.

We shouldn't be forcing links to be block elements using css.

One good reason is that the clickable area will span the width of the containing box rather than the text.

Reply:
Reverted back to the original logic

Reply:
👍